app/components/candidate_interface/continuous_applications/course_fee_row_helper.rb

```diff
@@ -0,0 +1,36 @@
+module CandidateInterface
+  module ContinuousApplications
+    module CourseFeeRowHelper
+      def course_fee_row
+        {
+          key: I18n.t('course_fee_row_helper.course_fee'),
+          value: domestic_fee + international_fee,
+        }
```

Comment:
Quick question: should we render the fees when the course is salary or apprenticeship?

Reply:
Good question, I'll ask Laura as the ticket does not cover that

Reply:
Don't show that row if it's not applicable. Everything else looks good.